pycaching/cache.py

```diff
@@ -624,6 +695,16 @@ def load(self):
 
         self.location = Point.from_string(root.find(id="uxLatLon").text)
 
+        self.lat = str(root.find(id="ctl00_ContentBody_Location").find("a")).split("=")[2].split("&")[0]
+
+        self.lon = str(root.find(id="ctl00_ContentBody_Location").find("a")).split("=")[3].split("&")[0]
+
+        self.id = str(root.find(id="ctl00_ContentBody_GeoNav_logButton")).split("=")[3].split("&")[0]
+
+        self.stateprovince = str(root.find(id="ctl00_ContentBody_Location").text.split(",")[0].split(" ")[1])
```

Comment:
I appreciate your effort to parse a state and a country, but the second `split(" ")[1]` is problematic. On the website, there is a text "In STATE, COUNTRY". What if the STATE part consists of multiple words (for example – see the screenshot attached to another comment)?

Notwithstanding that, I would prefer not to solve this in pycaching, but if you need detailed address info in your application, try something like [Google Geocoding API](https://stackoverflow.com/a/6355022/570503) instead. The benefit would be, that you could geocode any point, not only the cache (for example multicache waypoints, parking places, etc.).

Therefore, I would also suggest deleting this + the country parsing (including the properties and setters of course).